src/geophires_x_client/geophires_x_result.py

```diff
@@ -140,7 +140,7 @@ class GeophiresXResult:
                 # TODO moved to power generation profile, parse from there
                 #  'Annual Thermal Drawdown (%/year)',
                 'Bottom-hole temperature',
-                'Well seperation: fracture height',  # TODO correct typo upstream
+                'Well separation: fracture height',
```

Comment:
Yes, and the documentation spells it the right way ("separation"), so technically, this is a fix - if a user was using the parameter with the documented spelling, it was being ignored - fixing this means it will actually be functioning correctly!